kotlin-inject-compiler/core/src/main/kotlin/me/tatarka/inject/compiler/Options.kt

```diff
@@ -4,19 +4,22 @@ private const val OPTION_GENERATE_COMPANION_EXTENSIONS = "me.tatarka.inject.gene
 private const val OPTION_ENABLE_JAVAX_ANNOTATIONS = "me.tatarka.inject.enableJavaxAnnotations"
 private const val OPTION_USE_CLASS_REFERENCE_FOR_SCOPED_ACCESS = "me.tatarka.inject.useClassReferenceForScopeAccess"
 private const val OPTION_DUMP_GRAPH = "me.tatarka.inject.dumpGraph"
+private const val OPTION_GENERATE_ACTUAL_CREATE_FUNCTION = "me.tatarka.inject.generateActualCreateFunction"
```

Comment:
The naming is not great because "actual" is ambiguous, and can be read a few different ways.